environment.conf.json

```diff
@@ -67,7 +67,7 @@
       "zoom": 10,
       "maxZoom": 14,
       "minZoom": 8,
-      "gemeentenaam": "amsterdam"
+      "municipality": "amsterdam"
```

Comment:
I'm documenting the configuration properties and I think that we should take this prop out of the `options` context and should put it directly under `map` instead of `map.options`. The `options` prop should only contain Leaflet Map configuration options (https://leafletjs.com/reference-1.6.0.html#map-factory)